src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -63,8 +63,13 @@ protected override void Generate(MigrationOperation operation, IModel model, Mig
             CreateTableOperation operation,
             IModel model,
             MigrationCommandListBuilder builder,
-            bool terminate)
+            bool terminate = true)
         {
+            if (!terminate && operation.Comment != null)
+            {
+                throw new ArgumentException($"When generating migrations SQL for {nameof(CreateTableOperation)}, can't produce unterminated SQL with comments");
```

Comment:
What does it mean?

Reply:
This is just an internal sanity check - it's never supposed to happen.

The migrations SQL generator has an "architecture" where it gets the `terminate` flag, in order to allow appending more stuff at the end of the statement. But if there's a comment, then the line must be terminated because the comment setting happens in a different line...

Reply:
Okay, thanks!